### PR TITLE
Add theme fontsize

### PR DESCRIPTION
**Pull request by adamska27**
https://trello.com/c/BfD1f6xN/1130-revoir-la-font-de-lapp-pour-coh%C3%A9rence

## Detailed purpose of the PR

- hard coded font size
- design disagrement

### Result and observation

- add a theme for the font size
- update components to use new font size theme
- minor design fixes

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra dependency ?**
      If checked, Which extra dependency did you add ? (name, purpose, link ...).

### Testing Strategy

- [ ] Already covered by tests
- [x] Manual testing
- [x] Unit testing

### CODE REVIEW

File: src/modules/theme.js [hunk 1]
```diff
@@ -20,6 +20,8 @@ export type Colors = {
 
 export type FontWeightType = 'bold' | 'semiBold' | 'regular';
 
+export type FontSizeType = 'extraSmall' | 'small' | 'medium' | 'regular' | 'large' | 'extraLarge' | 'huge';
```
Comment (Fausto95) [inline src/modules/theme.js]: huge 😂

Comment (adamska27) [reply src/modules/theme.js]: on a beaucoup brainstormé avec Bertrand pour trouvé ça

File: src/types.js [hunk 2]
```diff
@@ -16,10 +16,9 @@ export type MimeType =
   | 'application/pdf'
   | 'application/vimeo';
 
-export type SpaceType = 'micro' | 'tiny' | 'small' | 'base' | 'large' | 'xlarge';
+export type SpaceType = 'micro' | 'tiny' | 'small' | 'base' | 'basePlus' | 'large' | 'xlarge';
```
Comment (VincentCATILLON) [inline src/types.js]: Can you please remove this value as it's not in UI guide

File: src/modules/theme.js [hunk 3]
```diff
@@ -58,6 +70,7 @@ const theme: Theme = {
     tiny: 8,
     small: 16,
     base: 24,
+    basePlus: 32,
```
Comment (VincentCATILLON) [inline src/modules/theme.js]: Same as above

File: src/modules/theme.js [hunk 4]
```diff
@@ -72,6 +85,15 @@ const theme: Theme = {
...
     bold: '700'
+  },
+  fontSize: {
+    extraSmall: 10,
+    small: 12,
+    medium: 13,
+    regular: 15,
+    large: 17,
+    extraLarge: 22,
+    huge: 28
```
Comment (VincentCATILLON) [inline src/modules/theme.js]: 👍

File: src/components/question.js [hunk 5]
```diff
@@ -40,7 +40,8 @@ export type State = {|
 
 const styles = StyleSheet.create({
   container: {
-    paddingVertical: theme.spacing.base,
+    paddingTop: theme.spacing.basePlus,
```
Comment (VincentCATILLON) [inline src/components/question.js]: please sum theme.spacing.base + theme.spacing.tiny please as it's only used there

**Pull request merged**